modules/client-handler/src/integrationTest/java/org/apache/ignite/client/handler/TestServer.java

```diff
@@ -49,12 +52,15 @@ public class TestServer {
 
     private final TestSslConfig testSslConfig;
 
+    private final AuthenticationConfiguration authenticationConfiguration;
+
     private final ClientHandlerMetricSource metrics = new ClientHandlerMetricSource();
 
     private long idleTimeout = 5000;
 
-    TestServer(@Nullable TestSslConfig testSslConfig) {
+    TestServer(@Nullable TestSslConfig testSslConfig, AuthenticationConfiguration authenticationConfiguration) {
```

Comment:
Can we make `authenticationConfiguration` nullable? It is not convenient to inject it in all tests where we don't actually need it.